cypress/globals.d.ts

```diff
@@ -1,5 +1,7 @@
 type Matcher = '$' | '^' | '~' | '*' | '';
 
+type Verbs = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH' | 'WATCH' | 'LIST' | 'CREATE' | 'UP'
```

Comment:
This may probably be convenient to have globally in the `@shell/types` maybe as `api.d.ts`.